package.json

```diff
@@ -18,7 +18,7 @@
 		"url": "https://github.com/LiskHQ/lisk-sdk/issues"
 	},
 	"engines": {
-		"node": ">=8.10 <=10",
```

Comment:
This range is different from the others, is this on purpose?

Reply:
Fixed based on Shu's comment too.